internal/cmd/status.go

```diff
@@ -63,29 +60,60 @@ func runStatus(cmd *cobra.Command, args []string) error {
 
 	runnerMgr := runner.NewManager(clusterMgr)
 
-	if len(args) == 1 {
-		// Show specific runner status
+	// Check if specific runner name was provided
+	if len(args) > 0 {
```

Comment:
This case should be handles by the same code that handles the list. Names should just be a single name in the case when an arg is passed.